hkmc2/shared/src/test/mlscript/ups/specialization/SimpleLiterals.mls

```diff
@@ -22,6 +22,12 @@ fun checkZeroOneTwo(x) = if x is
 //│ ╔══[ERROR] Unrecognized pattern split (annotated).
 //│ ║  l.14: 	  @compile Two then 2
 //│ ╙──      	   ^^^^^^^^^^^^^^^^^^
+//│ ╔══[WARNING] This else clause makes the following branches unreachable.
+//│ ╟── This branch is unreachable.
```

Comment:
🤔

Reply:
Fixed in c58b4fd94a659509aae892d50ef2fdcf6d61ec55.